src/components/Person/Person.jsx

```diff
@@ -1 +1,27 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age = null,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age ? <p className="Person__age">{`I am ${age}`}</p> : null}
+      <p className="Person__partner">
+        {isMarried ? (
+          `${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`
```

Comment:
sex === 'm' ? 'wife' : 'husband' - more correctly move this condition to separate variable. It will be cleaner